### PR TITLE
Improve syntax of Error Messages

### PR DESCRIPTION
**Pull request by jason-fox**
Update error messages to improve clarity

### CODE REVIEW

File: src/lib/orionld/serviceRoutines/orionldPostEntities.cpp [hunk 1]
```diff
@@ -220,13 +220,13 @@ static bool payloadCheck
...
   {
-    orionldErrorResponseCreate(ciP, OrionldBadRequestData, "No 'id' of the entity", "The 'id' field is mandatory", OrionldDetailsString);
+    orionldErrorResponseCreate(ciP, OrionldBadRequestData, "Entity id is missing", "The 'id' field is mandatory", OrionldDetailsString);
     return false;
   }
 
   if (typeNodeP == NULL)
   {
-    orionldErrorResponseCreate(ciP, OrionldBadRequestData, "No type of the entity", "The type field is mandatory", OrionldDetailsString);
+    orionldErrorResponseCreate(ciP, OrionldBadRequestData, "Entity type missing", "The type field is mandatory", OrionldDetailsString);
```
Comment (kzangeli) [inline src/lib/orionld/serviceRoutines/orionldPostEntities.cpp]: For consistency (with 'Entity id is missing'), add 'is'

Comment (jason-fox) [reply src/lib/orionld/serviceRoutines/orionldPostEntities.cpp]: Fixed: 17ee16d13

**Comment by kzangeli** [comment]
LGTM.
Let's wait to see the result from travis (if travis works ...)

**Comment by kzangeli** [comment]
Travis is not working, merging anyway

**Comment by kzangeli** [comment]
[ The functests failing due to this change I'll fix in my next PR ]

**Pull request merged**